src/main/thrift/parquet.thrift

```diff
@@ -604,8 +604,13 @@ struct PageHeader {
   /** Compressed page size in bytes (not including this header) **/
   3: required i32 compressed_page_size
 
-  /** 32bit crc for the data below. This allows for disabling checksumming in HDFS
-   *  if only a few pages needs to be read
+  /** The 32bit CRC for the page, to be be calculated as follows:
+   * - Using the standard CRC32 algorithm
+   * - On the data only, i.e. this header should not be included
+   * - On the compressed version of the data (unless no compression scheme is
```

Comment:
V2 pages handle RL and DL separately from the data (which term is usually used for the values only and not the levels). They are not compressed only encoded. (See DataPageHeaderV2 for more details.)
You shall be more specific about the RL and DL if they are included in the crc or not.

Reply:
@gszadovszky Thanks for the review, that was indeed an oversight of me. I've made the comment more specific to address this.